packages/riverpod_graph/test/integration/consumer_widget/golden/pubspec.yaml

```diff
@@ -0,0 +1,38 @@
+name: consumer_widget
+description: A new Flutter project.
+
+publish_to: "none"
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.14.0 <3.0.0"
+  flutter: ">=1.17.0"
+
+dependencies:
+  cupertino_icons: ^1.0.2
+  flutter:
+    sdk: flutter
+  flutter_riverpod:
+    path: ../../../../../../packages/flutter_riverpod
+  hooks_riverpod:
+    path: ../../../../../../packages/hooks_riverpod
+  riverpod:
+    path: ../../../../../../packages/riverpod
+
+dev_dependencies:
+  custom_lint: ^0.0.7
```

Comment:
Why is this a dependency?

Reply:
Oops, I removed it in https://github.com/rrousselGit/riverpod/pull/1581/commits/432f572e119383eced0de6435f7408b7f71eaa5a